OpenShift SRIOV network operator deployment failed on K8s cluster with Ubuntu 18.04 OS

Deployment of OpenShift SRIOV network operator failed on K8s cluster with Ubuntu OS with error in **Makefile**.

Steps to reproduce:

1. Install K8s cluster
2. Install GO on mater node with
    `snap install go --classic`
3. Download repo:
     ```
      mkdir /sriov 
      cd /sriov
      export GOPATH=/sriov
      go get github.com/openshift/sriov-network-operator
      ```
4. Start operator deployment

```
export PATH=$GOPATH/bin:$PATH
snap install jq
snap install skopeo --edge --devmode
cd /sriov/src/github.com/openshift/sriov-network-operator/
make deploy-setup-k8s
```
Installation stdout:

```
go: creating new go.mod: module tmp
go: found sigs.k8s.io/controller-tools/cmd/controller-gen in sigs.k8s.io/controller-tools v0.3.0
go: downloading k8s.io/apimachinery v0.18.2
go: downloading github.com/spf13/cobra v0.0.5
go: downloading gopkg.in/yaml.v3 v3.0.0-20190905181640-827449938966
go: downloading k8s.io/apiextensions-apiserver v0.18.2
go: downloading golang.org/x/tools v0.0.0-20190920225731-5eefd052ad72
go: downloading k8s.io/api v0.18.2
go: downloading sigs.k8s.io/yaml v1.2.0
go: downloading github.com/fatih/color v1.7.0
go: downloading github.com/spf13/pflag v1.0.5
go: downloading github.com/mattn/go-colorable v0.1.2
go: downloading github.com/inconshreveable/mousetrap v1.0.0
go: downloading github.com/gogo/protobuf v1.3.1
go: downloading k8s.io/utils v0.0.0-20200324210504-a9aa75ae1b89
go: downloading github.com/google/gofuzz v1.1.0
go: downloading k8s.io/klog v1.0.0
go: downloading github.com/mattn/go-isatty v0.0.8
go: downloading github.com/gobuffalo/flect v0.2.0
go: downloading gopkg.in/inf.v0 v0.9.1
go: downloading sigs.k8s.io/structured-merge-diff/v3 v3.0.0
go: downloading golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
go: downloading gopkg.in/yaml.v2 v2.2.8
go: downloading golang.org/x/sys v0.0.0-20191022100944-742c48ecaeb7
go: downloading github.com/json-iterator/go v1.1.8
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/modern-go/reflect2 v1.0.1
go: downloading golang.org/x/text v0.3.2
/sriov/bin/controller-gen "crd:crdVersions={v1},trivialVersions=true" webhook paths="./..." output:crd:artifacts:config=config/crd/bases
go: creating new go.mod: module tmp
go: downloading sigs.k8s.io/kustomize/kustomize/v3 v3.5.4
go: downloading sigs.k8s.io/kustomize v2.0.3+incompatible
go: downloading sigs.k8s.io/kustomize/api v0.3.2
go: downloading k8s.io/client-go v0.17.0
go: downloading github.com/pkg/errors v0.8.1
go: downloading sigs.k8s.io/yaml v1.1.0
go: downloading sigs.k8s.io/kustomize/cmd/config v0.0.5
go: downloading gopkg.in/yaml.v2 v2.2.4
go: downloading k8s.io/api v0.17.0
go: downloading k8s.io/apimachinery v0.17.0
go: downloading sigs.k8s.io/kustomize/kyaml v0.0.6
go: downloading github.com/posener/complete/v2 v2.0.1-alpha.12
go: downloading github.com/evanphx/json-patch v4.5.0+incompatible
go: downloading github.com/google/gofuzz v1.0.0
go: downloading github.com/olekukonko/tablewriter v0.0.4
go: downloading github.com/go-errors/errors v1.0.1
go: downloading k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
go: downloading github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
go: downloading github.com/Azure/go-autorest/autorest v0.9.0
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading github.com/googleapis/gnostic v0.0.0-20170729233727-0c5108395e2d
go: downloading golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
go: downloading sigs.k8s.io/kustomize/cmd/kubectl v0.0.3
go: downloading github.com/mattn/go-runewidth v0.0.7
go: downloading github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
go: downloading github.com/Azure/go-autorest/autorest/adal v0.5.0
go: downloading github.com/xlab/treeprint v0.0.0-20181112141820-a009c3971eca
go: downloading github.com/hashicorp/go-multierror v1.0.0
go: downloading gopkg.in/yaml.v3 v3.0.0-20191026110619-0b21df46bc1d
go: downloading github.com/gophercloud/gophercloud v0.1.0
go: downloading k8s.io/kubectl v0.0.0-20191219154910-1528d4eea6dd
go: downloading github.com/go-openapi/spec v0.19.5
go: downloading k8s.io/cli-runtime v0.17.0
go: downloading google.golang.org/appengine v1.5.0
go: downloading github.com/hashicorp/errwrap v1.0.0
go: downloading github.com/golang/protobuf v1.3.2
go: downloading github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading github.com/go-openapi/jsonreference v0.19.3
go: downloading github.com/go-openapi/swag v0.19.5
go: downloading golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392
go: downloading github.com/imdario/mergo v0.3.5
go: downloading github.com/Azure/go-autorest/autorest/date v0.1.0
go: downloading github.com/Azure/go-autorest/tracing v0.5.0
go: downloading github.com/mailru/easyjson v0.7.0
go: downloading cloud.google.com/go v0.38.0
go: downloading k8s.io/utils v0.0.0-20191114184206-e782cd3c129f
go: downloading github.com/PuerkitoBio/purell v1.1.1
go: downloading github.com/go-openapi/jsonpointer v0.19.3
go: downloading github.com/emicklei/go-restful v2.9.5+incompatible
go: downloading github.com/exponent-io/jsonpath v0.0.0-20151013193312-d6023ce2651d
go: downloading github.com/chai2010/gettext-go v0.0.0-20160711120539-c6fed771bfd5
go: downloading github.com/Azure/go-autorest/logger v0.1.0
go: downloading k8s.io/component-base v0.17.0
go: downloading github.com/mitchellh/go-wordwrap v1.0.0
go: downloading github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578
go: downloading github.com/jonboulle/clockwork v0.1.0
go: downloading github.com/ghodss/yaml v1.0.0
go: downloading golang.org/x/sys v0.0.0-20190922100055-0a153f010e69
go: downloading golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: downloading github.com/docker/docker v0.7.3-0.20190327010347-be7ac8be2ae0
go: downloading github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7
go: downloading github.com/russross/blackfriday v1.5.2
go: downloading github.com/docker/spdystream v0.0.0-20160310174837-449fdfce4d96
go: downloading github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd
go: downloading github.com/posener/script v1.0.4
go: downloading github.com/liggitt/tabwriter v0.0.0-20181228230101-89fcab3d43de
go: downloading github.com/peterbourgon/diskv v2.0.1+incompatible
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/google/go-cmp v0.3.0
go: downloading github.com/google/btree v1.0.0
go: downloading github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78
go: downloading github.com/sirupsen/logrus v1.4.2
go: downloading github.com/konsorten/go-windows-terminal-sequences v1.0.1
/sriov//bin/kustomize build config/crd | kubectl apply -f -
customresourcedefinition.apiextensions.k8s.io/sriovibnetworks.sriovnetwork.openshift.io created
customresourcedefinition.apiextensions.k8s.io/sriovnetworknodepolicies.sriovnetwork.openshift.io created
customresourcedefinition.apiextensions.k8s.io/sriovnetworknodestates.sriovnetwork.openshift.io created
customresourcedefinition.apiextensions.k8s.io/sriovnetworks.sriovnetwork.openshift.io created
customresourcedefinition.apiextensions.k8s.io/sriovoperatorconfigs.sriovnetwork.openshift.io created
/bin/sh: 1: Syntax error: "(" unexpected
Makefile:169: recipe for target 'deploy-setup' failed
make: *** [deploy-setup] Error 2

```

--- a/Makefile
+++ b/Makefile
@@ -166,7 +166,7 @@ bundle-build:
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
 deploy-setup: install
-	@EXCLUSIONS=() hack/deploy-setup.sh $(NAMESPACE)
+	hack/deploy-setup.sh $(NAMESPACE)
 
 deploy-setup-k8s: export NAMESPACE=sriov-network-operator
 deploy-setup-k8s: export ENABLE_ADMISSION_CONTROLLER=false

